### PR TITLE
Update favicon

### PR DESCRIPTION
**Pull request by omergulen**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-
+<head>
+  <link rel="shortcut icon" type="image/png" href="https://d30y9cdsu7xlg0.cloudfront.net/png/83825-200.png"/>
```
Comment (anthonydillon) [inline index.md]: This won't be valid markup without a `title`.

Comment (filfreire) [reply index.md]: @omergulen thanks for the contribution. I'd like to add to @anthonydillon comment another request, the href should be a link to the actual file in the repository [here](images/icon/favicon.ico).

**Review by filfreire — CHANGES_REQUESTED** [review]
@omergulen thanks for the contribution. I'd like to add to @anthonydillon comment another request, the href should be a link to the actual file in the repository https://github.com/PortoTestersMeetup/portotestersmeetup.github.io/blob/master/images/icon/favicon.ico.

**Comment by FabioCB** [comment]
@omergulen thank you for your contribution. We're merging another PR that solves this issue and closing yours. Feel free to contribute on other [issues](https://github.com/PortoTestersMeetup/portotestersmeetup.github.io/issues)

**Pull request closed without merging**